src/main/scala/DemoFortis.scala

```diff
@@ -173,6 +181,66 @@ object DemoFortis {
       }
     }
 
+    if (mode.contains("tadaweb")) {
+      streamProvider.buildStream[TadawebEvent](ssc, streamRegistry("tadaweb")) match {
+        case Some(stream) => stream
+          .map(event => AnalyzedItem(
+            event,
+            source = event.tada.name,
+            analysis = Analysis(keywords =
+              keywordExtractor.extractKeywords(event.text) ::: keywordExtractor.extractKeywords(event.title)
+            )
+          ))
+          .filter(_.analysis.keywords.nonEmpty)
+          .map(fortisEvent => {
+            val language = languageDetection.detectLanguage(fortisEvent.originalItem.text)
+            val sentiment: Option[Double] = fortisEvent.originalItem.sentiment match {
+              case "negative" => Some(0)
+              case "neutral" => Some(0.6)
+              case "positive" => Some(1)
+              case _ => language match {
+                case Some(lang) =>
+                  if (supportedLanguages.contains(lang))
```

Comment:
Removed check. Since the sentiment detector is able to handle the unsupported case, it seems cleaner to me to filter as the next stage after attempting to detect sentiment. For Tadaweb, the sentiment can come from the event itself. Filtering in this way avoids duplicating that check.